### DISCUSSION TITLE
添加获取消息的API

### DISCUSSION THREAD

**Discussion by Fgaoxing** [Qexo/Qexo#111] [General]
添加获取消息的API
添加Qexo版本获取API

**Comment by Fgaoxing** [comment]
API修改的文件标签应该为  Del by Qexo-API

**Reply by am-abudu** [reply]
b9a3867c94030240e4e8069f6f7e7bb77064daac

**Reply by Fgaoxing** [reply]
@am-abudu 我说的是对外API，内部的不需要啊！

**Reply by am-abudu** [reply]
> @am-abudu 我说的是对外API，内部的不需要啊！

均已添加 请查看完整commit

**Reply by Fgaoxing** [reply]
@am-abudu 我指的的是api.py中内部的api（api目录下的）是Del by Qexo
而pub.py中的对外api(pub目录下的)的是Del by Qexo-API

**Reply by am-abudu** [reply]
> @am-abudu 我指的的是api.py中内部的api（api目录下的）是Del by Qexo 而pub.py中的对外api(pub目录下的)的是Del by Qexo-API

我已对此进行区分 **请看完整COMMIT再来回复**

**Comment by am-abudu** [comment]
7a2cf7a721668c2c51e24d8d403e2988cc401b4f
